docs/resources/user_role_binding.md

```diff
@@ -3,14 +3,79 @@
 subcategory: ""
 description: |-
-  This resource is under development and is not generally available yet.
+  Provides a Lightstep Role Binding https://api-docs.lightstep.com/reference/RoleBinding. This can be used to manage User's Organization level roles and Project level roles.
+  A role binding can target either the Organization level roles or a Project role for a specific project. Users can't be assigned to a more restrictive Project role than their Organization level role.
+  NOTE: this terraform resource is authoritative, users that are not declared in a terraform resource will lose the declared role in the specified organization/project.
+  The list of valid roles for Organization level role bindings are:
+  * Organization Admin
+  * Organization Editor
+  * Organization Viewer
+  The list of valid roles for Project level role bindings are:
+  * Project Editor
+  * Project Viewer
+  Changes to both Organization level roles and Project level roles for the same user can cause race conditions,
+  in that case, we suggest these changes be made in two steps.
+  * When lowering a user's Organization level role and upping their Project level Role, first change their organization role.
+  * When upping a user's Organization level role and removing or lowering their Project level Role, first change their project role.
 ---
 
 # lightstep_user_role_binding (Resource)
 
-This resource is under development and is not generally available yet.
+Provides a [Lightstep Role Binding](https://api-docs.lightstep.com/reference/RoleBinding). This can be used to manage User's Organization level roles and Project level roles.
 
+A role binding can target either the Organization level roles or a Project role for a specific project. Users can't be assigned to a more restrictive Project role than their Organization level role. 
 
+**NOTE**: this terraform resource is authoritative, users that are not declared in a terraform resource will lose the declared role in the specified organization/project.
+
+The list of valid roles for Organization level role bindings are:
+* Organization Admin
+* Organization Editor
+* Organization Viewer
+
+
+The list of valid roles for Project level role bindings are:
+* Project Editor
+* Project Viewer
+
+
+Changes to both Organization level roles and Project level roles for the same user can cause race conditions, 
+in that case, we suggest these changes be made in two steps. 
+* When lowering a user's Organization level role and upping their Project level Role, first change their organization role.
+* When upping a user's Organization level role and removing or lowering their Project level Role, first change their project role.
+
+## Example Usage
+
+```terraform
+# The following example shows how to configure two users with the "Organization Restricted Member" and
+# override their permissions in "Project A", one with the "Project Editor" role and the other with the "Project Viewer" role.
```

Comment:
```suggestion
# It also overrides their permissions in "Project A", one with the "Project Editor" role and the other with the "Project Viewer" role.
```
Nit: Splits the sentences to lower the reading level because people skim